test/functional/tools/select_from_url.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="select_from_url" name="select_from_url" version="0.1.0" profile="23.1">
+    <command><![CDATA[
+echo '$url_param_value' > '$param_value' &&
+echo '$url_param_value_postprocessed' > '$param_value_postprocessed' &&
+echo '$invalid_url_param_value_postprocessed' > '$invalid_param_value_postprocessed'
+    ]]></command>
+    <inputs>
+        <param name="url_param_value" type="select">
+            <options from_url="https://usegalaxy.org/api/genomes">
```

Comment:
Wondering how `"dm6"` ends up as an option here. I would have expected the keys of the dictionary become the options here, e.g. `"id"`, ...?

Reply:
The response from https://usegalaxy.org/api/genomes is exactly what's consumed, so no postprocessing needed. Tests can use `dm6` or the actual displayed value to select something, here we're selecting by value (https://github.com/galaxyproject/galaxy/pull/17087/files#diff-c044b7c444d912e0d18e5c0c442927ef31fc557507450ac11264331fdbc67ee4R44). Does that answer the question ?